### PR TITLE
Neaf 82 migrate reports to postgresql

### PR DESCRIPTION
**Pull request by VladimirBaklan**

### CODE REVIEW

File: reports/day_of_week.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Day_of_week,Accident_count,Percentage
+4) Thu,529,100.0
```
Comment (sleshJdev) [inline reports/day_of_week.csv]: ?

**Review by sleshJdev — CHANGES_REQUESTED** [review]

File: reports/day_of_week.csv (hunk 1 above)
Comment (VladimirBaklan) [reply reports/day_of_week.csv]: Ooops. Gitignore updated :)

File: src/main/scala/com/itechart/ny_accidents/report/Reports.scala [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+  private lazy val DATE_COLUMN_NAME = "creation_date"
 
   def generateReportForTupleRDD[A <: Product](data: RDD[A], header: Seq[String]): Seq[Seq[String]] = {
     header +: data.collect().map(obj => obj.productIterator.toSeq.map(_.toString))
   }
 
+  def generateDataFrameReportForTupleRDD[A <: Product](data: RDD[A], schema: StructType): DataFrame = {
+    val rdd = data.map(Row.fromTuple)
+    Spark.sparkSql.createDataFrame(rdd, schema)
+      .withColumn(PK_DATAFRAME_NAME,monotonically_increasing_id())
```
Comment (sleshJdev) [inline src/main/scala/com/itechart/ny_accidents/report/Reports.scala]: Missed white space after comma, please format code.

**Review by sleshJdev — CHANGES_REQUESTED** [review]

File: src/main/scala/com/itechart/ny_accidents/report/Reports.scala (hunk 2 above)
Comment (VladimirBaklan) [reply src/main/scala/com/itechart/ny_accidents/report/Reports.scala]: Formatted

**Review by sleshJdev — APPROVED** [review]

**Review by AlexKostritsa — APPROVED** [review]

**Pull request merged**